CreeDictionary/tests/conftest.py

```diff
@@ -13,7 +13,7 @@
 
 from hypothesis import settings
 
-settings.register_profile("default", deadline=timedelta(milliseconds=500))
+settings.register_profile("default", deadline=timedelta(milliseconds=1000))
```

Comment:
When I started unit test with coverage. I got a DeadlineExceeded exception from hypothesis saying one function call took 800 ms. I think coverage slows things down, and also I use a garbage laptop in office so yeah.